contracts/colonyNetwork/ColonyNetworkSkills.sol

```diff
@@ -277,7 +263,28 @@ contract ColonyNetworkSkills is ColonyNetworkStorage, Multicall {
     }
   }
 
-  function addBridgedReputationUpdate(uint256 _chainId, address _colony) public stoppable onlyMiningChain {
+  function addPendingSkill(address _bridgeAddress, uint256 _skillId)
+    public
+    always
+    onlyMiningChain
+  {
+    uint256 bridgeChainId = bridgeData[_bridgeAddress].chainId;
+    require(bridgeChainId != 0, "colony-network-not-known-bridge");
```

Comment:
Should we check skillId/bridgeChainId consistency here as well?

Reply:
Sure! Potentially worth creating a modifier which does all the checks, since we do them elsewhere.